test/test_simplendf.py

```diff
@@ -23,12 +23,64 @@ def test_simpleread(self):
         self.assertEqual(indf.label, 'Signal')
         self.assertEqual(indf.units, 'counts')
         self.assertEqual(indf.title, 'Test Data')
+        self.assertEqual(indf.xnumb, 1 )
+        self.assertEqual(indf.dim, [5,3])
+
+        # Xtension sutf
+        # xnumb (property), xname, xloc,  xstat, xdel,  xnew
+        self.assertEqual(indf.xname(0), 'FITS')
+        self.assertEqual(indf.xstat('FITS'), True)
+        self.assertEqual(indf.xstat('NONEXISTENT'), False)
+        self.assertIsInstance(type(indf.xloc('FITS', 'READ')), type(hds.HDSWrapperClass))
```

Comment:
I'm confused by the `type()` calls here. `IsInstance` is meant to be checking instances and not classes.